factory/usr/lib/core/extra-paths

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -eu
+
+cat <<EOF >>/sysroot/etc/fstab
+/run/mnt/data /writable none bind,x-initrd.mount 0 0
```

Comment:
Isn't this what `sysroot-writable.mount` mounts? Shouldn't you remove it as for the kernel ones?

Reply:
`/etc/fstab` is bind mounted from `/run` by `handle-writable-paths`. So we have to run this script after `handle-writable-paths`. `handle-writable-paths` also needs `/writable` to be mounted. So that means this script depends on `sysroot-writable.mount`.